### ISSUE TITLE
Dialob button focus is not visible enough in High Contrast mode.

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor5#10542]
## 📝 Provide detailed reproduction steps (if any)

1.  Switch your screen to High Contrast (HC) Mode in white color.
2.  Open https://ckeditor.com/docs/ckeditor4/latest/examples/fullpreset.html
3.  Open link dialog and start tabbing through fields until you reach Ok/Cancel button

### ✔️ Expected result

When Ok or Cancel button is focued it should be clearly visible.

### ❌ Actual result

Focus is really subtle and could be missed by visually impaired users. 

![](https://user-images.githubusercontent.com/5314687/133578512-d1404253-faff-4a72-b5c8-27a952b6517c.gif)

### ❓ Possible solution

File `ckeditor/skins/moono-lisa/dialog.css` contains the following CSS rule

```
.cke_hc a.cke_dialog_ui_button {
    border: 3px solid #bcbcbc;
}
```

If you change `3px` to `1px`, it should  become visible enough.

![](https://user-images.githubusercontent.com/5314687/133581806-bbdff55a-ac20-4117-a41e-a88dba8a8169.gif)

## 📃 Other details

*   Browser: Any
*   OS: Windows
*   First affected CKEditor version: 4.6
*   Installed CKEditor plugins: N/A

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
